pages/linux/nsenter.md

```diff
@@ -4,14 +4,18 @@
 > Particularly useful for docker images or chroot jails.
 > More information: <https://github.com/jpetazzo/nsenter/>.
 
-- Run command in existing processes network namespace:
+- Run command using all the same namespaces as an existing process:
+
+`nsenter -t {{pid}} -a {{command}} {{command_arguments}}`
+
+- Run command in an existing process's network namespace:
```

Comment:
```suggestion
- Run a specific command in an existing process's network namespace:
```

Reply:
the same namespaces as an existing process:

Can we incorporate this, as it is used above.